.vib/solr/goss/goss.yaml

```diff
@@ -0,0 +1,39 @@
+command:
+  which-solr:
+    exec: which solr
+    exit-status: 0
+    stdout:
+      - "/opt/bitnami/solr/bin/solr"
+    stderr: [ ]
+    timeout: 40000
+  solr-version-ok:
+    exec: solr version | grep --quiet "$APP_VERSION"
+    exit-status: 0
+    stdout: [ ]
+    stderr: [ ]
+    timeout: 40000
+  solr-assert-running:
+    exec: SOLR_AUTH_TYPE=basic SOLR_AUTHENTICATION_OPTS="-Dbasicauth=${SOLR_ADMIN_USERNAME}:${SOLR_ADMIN_PASSWORD}" solr assert --cloud "http://localhost:${SOLR_PORT_NUMBER}/solr/"
```

Comment:
What does this command give as an output? Is it something we could take advantage of?

Reply:
This command ensures that the SolR cluster is running, and in fact, that is running under "cloud" mode (one of the SolR setups).